packages/backend/src/clients/Slack/Slackbot.ts

```diff
@@ -107,7 +116,9 @@ export class SlackService {
 
             try {
                 const { teamId } = context;
-                const imageId = `slack-image-${context.teamId}-${event.unfurl_id}-${index}`;
+                const imageId = `slack-image-${context.teamId}-${
```

Comment:
could we remove the information from the id? And just use a nano id? Is there any benefit to encoding info in the url?

Reply:
There are no extra benefits, I'll replace it with just the nanoid